fl4health/servers/scaffold_server.py

```diff
@@ -19,6 +20,7 @@ class ScaffoldServer(FlServer):
     def __init__(
         self,
         client_manager: ClientManager,
+        fl_config: Config,
```

Comment:
Yeah, I think this was an oversight for us, as it didn't exist in our implementation before. However, this will be a moot point, as the checkpointing work in [this PR](https://github.com/VectorInstitute/FL4Health/pull/298) adds it properly to this server class.

Reply:
Discussed and this is a moot point as its handled in the following PR